api/v1alpha1/lvmvolumegroup_types.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// LVMVolumeGroupSpec defines the desired state of LVMVolumeGroup
+type LVMVolumeGroupSpec struct {
+	// DeviceSelector is a set of rules that should match for a device to be included in this TopoLVMCluster
+	// +optional
+	DeviceSelector *DeviceSelector `json:"deviceSelector,omitempty"`
+
+	// NodeSelector chooses nodes
+	// +optional
+	NodeSelector *corev1.NodeSelector `json:"nodeSelector,omitempty"`
+
+	// TODO: add support for various LVM settings
+	// // Config for this deviceClass, lvm settings are a field here
+	// // +optional
+	// Config *DeviceClassConfig `json:"config,omitempty"`
+}
+
+// LVMVolumeGroupStatus defines the observed state of LVMVolumeGroup
+type LVMVolumeGroupStatus struct {
+	// NodeStatus contains the per node status of the VG
+	NodeStatus []NodeStatus `json:"nodeStatus,omitempty"`
+}
+
+type NodeStatus struct {
+	// Node is the name of the node where the VG is to be created
+	Node string `json:"node,omitempty"`
+	// Status tells if the VG was created on the node
+	Status string `json:"status,omitempty"`
+	// Reason provides more detail on the VG creation status
+	Reason string `json:"reason,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// LVMVolumeGroup is the Schema for the lvmvolumegroups API
```

Comment:
```suggestion
// LVMVolumeGroup is the schema for the lvmvolumegroups API
```

Reply:
This was autogenerated by operator-sdk. The same comment exists in the other type definitions as well.